src/features/events/components/EventRelatedCard.tsx

```diff
@@ -0,0 +1,49 @@
+import { FC } from 'react';
+import { Box, Divider } from '@mui/material';
+
+import { EventsModel } from '../models/EventsModel';
+import messageIds from 'features/events/l10n/messageIds';
+import RelatedEventCard from './LocationModal/RelatedEventCard';
+import { useMessages } from 'core/i18n';
+import { ZetkinEvent } from 'utils/types/zetkin';
+import ZUICard from 'zui/ZUICard';
+import ZUIFuture from 'zui/ZUIFuture';
+
+interface EventRelatedCardProps {
+  data: ZetkinEvent;
+  model: EventsModel;
+}
+
+const EventRelatedCard: FC<EventRelatedCardProps> = ({ data, model }) => {
+  const messages = useMessages(messageIds);
+
+  return (
+    <Box mt={2}>
+      <ZUICard header={messages.eventRelatedCard.header()}>
+        <ZUIFuture
+          future={model.getParallelEvents(data.start_time, data.end_time)}
```

Comment:
This is not the right function to use to get the related events. The issue states that the following events should be included in this list:

* Events that happen just before, or just after the current event
* Events that happen exactly in parallel, i.e. same start and end time, with same event type

These are not the events returned by `getParallelEvents()`. That method returns all events on the same days as the original event. You can use `getParallelEvents()` as an inspiration, but you need to write a new method to reflect the logic required by the issue.

I've updated the issue to uncheck those two requirements so that it's clear what I think is missing.